common/lib/xblock/__init__.py

```diff
@@ -0,0 +1,10 @@
+'''
```

Comment:
FYI at the conference today we discussed what it would take to move the XBlock into its own repository. It might not be too difficult to move just the new code, but it would be a larger undertaking to move all of the client-side code too. Also, it would be hard to move the discussion board (aka the tab) and the team's discussion. Given this, we agreed that it seems simplest to keep it in edx-platform for now, and just move it to openedx.